pubspec.yaml

```diff
@@ -12,20 +12,20 @@ dependencies:
     sdk: flutter
   package_info: ^0.4.0+18
   chewie: ^0.9.10
-  video_player: ^0.10.11
-  flutter_widget_from_html: ^0.3.3+3
+  video_player: ^0.10.11+1
+  flutter_widget_from_html: ^0.3.3+4
   matrix_gesture_detector: ^0.1.0
-  url_launcher: ^5.4.7
+  url_launcher: ^5.4.10
```

Comment:
Do we need to manually change the minor version? ^5.4.7 is equivalent to '>=5.4.7 <6.0.0' (next non-broken version). I think if we want to upgrade the dependency version, we can just do `flutter pub upgrade`

Reply:
No we don't as the `^` does that for us. I just do this as a manual check ie. I check the newer releases and see what has changed, so I can figure out what I might need to re-test.

So the manual update of minor version is just so we know we've looked at the changes up to this minor version and they seem ok for this project. Otherwise yes we can always do a `flutter pub upgrade` and so the versions in use are actually the ones in `pubspec.lock`